WhaleSpotting/Repositories/LikeRepo.cs

```diff
@@ -0,0 +1,30 @@
+using WhaleSpotting.Models.Database;
+using WhaleSpotting.Models.Request;
+
+namespace WhaleSpotting.Repositories;
+
+public interface ILikeRepo
+{
+    public void Create(LikeRequest newLikeRequest);
+}
+
+public class LikeRepo : ILikeRepo
+{
+    private readonly WhaleSpottingDbContext context;
+
+    public LikeRepo(WhaleSpottingDbContext context)
+    {
+        this.context = context;
+    }
+    
+    public void Create(LikeRequest newLikeRequest)
+    {
+        var insertResult = context.Likes.Add(new Like
+        {
+            Date = DateTime.Now.ToUniversalTime(),
+            WhaleSightingId = newLikeRequest.WhaleSightingId,
+            UserId = newLikeRequest.UserId,
+        });
+        context.SaveChanges();
+    }
+}
```

Comment:
Standard practice is to end each file with exactly one blank line - see how GitHub has added a ⛔ symbol here?